### PR TITLE
Update to v5

### PR DESCRIPTION
**Pull request by khalford**
### Description:

Update to version 5 of the chatops image. This gives us access to the `/find-host` comand. For this we need to add the node IP to the environment variables and add the pull requests channel to the config.

---

### Submitter:

Have you:

* [x] Labelled this PR, e.g. `bug`, `deployment`, `enhancement` ...etc.

- A `deployment` can be reviewed, and merged, by a single reviewer.
- It can only be used to deploy, change, or remove clusters based on existing patterns for staging.
- Anything involving prod, or production facing services must use the normal 2 person review.
- All other PR types require the usual PR process (e.g. 2 person).

---

### Reviewer

Have you:

* [ ] Verified this PR uses the correct label(s) based on the rules above?
* [ ] Checked if this could affect production (e.g. a global value that's changed without an override)?
* [ ] Tested setting this up, if it's not a deployment, to verify it can be redeployed with any documentation if appropriate?

### CODE REVIEW

File: charts/dev/chatops/values.yaml [hunk 1]
```diff
@@ -23,3 +23,4 @@ repos:
     - ansible-jupyter
     - SCD-Openstack-Utils
     - check-version-action
+channel: C03RT2F6WHZ
```
Comment (anish-mudaraddi) [inline charts/dev/chatops/values.yaml:26]: Should this be a secret?

File: charts/dev/chatops/templates/cloud-chatops.yaml [hunk 2]
```diff
@@ -21,6 +21,11 @@ spec:
       containers:
       - name: {{ .Values.app }}
         image: {{ .Values.image.repository }}:{{ .Values.image.tag}}
+        env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
```
Comment (anish-mudaraddi) [inline charts/dev/chatops/templates/cloud-chatops.yaml:28]: what's this for?

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
have some questions

File: charts/dev/chatops/values.yaml (hunk 1 above)
Comment (khalford) [reply charts/dev/chatops/values.yaml:26]: Not really, it is just a channel ID. It is only accessible from inside our workspace so I see no issue with it being public.

File: charts/dev/chatops/templates/cloud-chatops.yaml (hunk 2 above)
Comment (khalford) [reply charts/dev/chatops/templates/cloud-chatops.yaml:28]: This is for pod to access its node IP address. This is used in one of the slash commands in the app.

**Review by anish-mudaraddi — APPROVED** [review]

**Pull request merged**